InMobiAdsDemo/InMobiAdsDemoTests/InMobiAdsDemoTests.m

```diff
@@ -0,0 +1,39 @@
+//
```

Comment:
no test cases have been mentioned. Please add test cases or remove all test cases related classes, if not needed.

Reply:
Removed all of them as they previously also didn't existed .